resources/default_pipeline_environment.yml

```diff
@@ -601,14 +601,12 @@ steps:
       verbose: false
   uiVeri5ExecuteTests:
     failOnError: false
-    dockerEnvVars: {}
     installCommand: 'npm install @ui5/uiveri5 --global --quiet'
-    seleniumPort: 4444
     stashContent:
       - 'buildDescriptor'
       - 'tests'
-    testOptions: ''
-    runCommand: "uiveri5 --seleniumAddress='http://${config.seleniumHost}:${config.seleniumPort}/wd/hub'"
+    runCommand: "/home/node/.npm-global/bin/uiveri5"
```

Comment:
The defaults can be omitted here since they now come with the golang based step (defined in step.yaml)